### ISSUE TITLE
Dynamic Inline Styles 

### ISSUE THREAD

**Issue by jdpagley** [jpuri/react-draft-wysiwyg#118]
Right now we generate width and height of our editor dynamically. This presents a problem because there are way too many size variations to solve with pre-defined classes. It would be nice if we could set our dynamic inline styles directly on the different components of the editor.

It would be nice to have something like:

```
   wrapperStyles={...} 
   editorStyles={{height: '300px', width: '500px'}} 
   toolbarStyles={...}
```

**Comment by jpuri** [comment]
Hey @jdpagley,
I see an issue here DraftJS does not supports style property on editor component. There are some libs that create classes out of style objects that might be an option for you.

Plz suggest if you have some other idea of dealing with it.

**Comment by jdpagley** [comment]
@jpuri you're right that does seem to be difficult with current draftjs. No worries :)

**Comment by jpuri** [comment]
I think it would anyways be nicer to have style property available for editor wrapper. In fact with styling of drafts editor to to take 100% width and height of container above requirement can also be achieved.

I will add these style props to the editor component. Prioritizing it for milestone `1.6.0`.

**Comment by jpuri** [comment]
@jdpagley: I have added style properties: 

```
    toolbarStyle: PropTypes.object,
    editorStyle: PropTypes.object,
    wrapperStyle: PropTypes.object,
```

Since draftJS editor takes 100% these will be helpful in changing editor width.

Change is in develop branch I will release it very soon.

**Comment by jpuri** [comment]
Change released in version `react-draft-wysiwyg@1.5.7` 😄
